Created branch to add Works and About me section in Microverse Portfolio

steveWDamesJr opened the pull request:
Add:

- Images for works and about me section
- Works section
- About me section
- Hover | Pressed buttons for sections
- Implement Grid for works and about section layout
- Implement Flexbox to style content layout
- Styles to Work & About Me sections (CSS)
- Debugged HTML & CSS

steveWDamesJr committed:
feat: add works section card to markup

Add:
-Works and About Me section images from Figma template
-Add first Work project Tonic Card to markup
-Add linters

steveWDamesJr committed:
Add Multi-Post stories card 1 to markup

steveWDamesJr committed:
Add Tonic Project card 2 to markup

steveWDamesJr committed:
Add Multi Post Stories card 2 to markup

steveWDamesJr committed:
Add About Me card to markup

steveWDamesJr committed:
Update markup and stylesheet

Feat:
-Add Works section to markup with grid structure in stylesheet
-Refactor Homepage: remove padding and restructure
-Add about me section to markup with hero title description CTA social media icons button and languages: Add flexbox structure
Add languages images
Fix: linter errors

steveWDamesJr committed:
Update wrapper margin and card colors

steveWDamesJr committed:
Add button hover and focus effects for works and About sec

julie-ify commented on index.html:
```diff
@@ -15,38 +15,224 @@
+      </ul>
+    </section>
+
+    <div class="works-section-grid">
```
- [x] Your project looks amazing 💯 and I admire it. However, for better code organization and HTML best practices, please consider using a `section` tag instead of `div` tag to improve user interaction and better semantic HTML 😃.

julie-ify commented on index.html:
```diff
@@ -15,38 +15,224 @@
+        </div>
+      </article>
+
+
+      <article class="card2">
```
- [x] For better code organization and HTML best practices, please consider using a `section` tag instead of `article` tag to improve user interaction and for better semantic HTML.

julie-ify commented on index.html:
```diff
@@ -15,38 +15,224 @@
-    </section>
-  </div>
+        <div class="skills-list">
+          <a class="card-button2" href="#">Get my resume</a>
+
+
+
+
+
+
+        </div>
```
- [x] Consider removing these empty lines `184 - 189` to improve your code structure and to make your work look professional 😃

julie-ify commented on index.html:
```diff
@@ -15,38 +15,224 @@
+
+        </div>
+
+        <article class="All-skills">
+          <div class="lang-container">
+            <div class="lang-arrow">
+              <h3>Languages</h3>
+              <img src="./images/arrow1.png" alt="Arrow down">
+            </div>
+            <ul class="program-lang">
+              <li class="programs">
+                <img src="./images/js.png" alt="javaScript logo">
+                <p class="program-name">JavaScript</p>
+              </li>
+              <li class="programs">
+                <img src="./images/html.png" alt="HTML logo">
+                <p class="program-name">HTML</p>
+              </li>
+              <li class="programs">
+                <img src="./images/css.png" alt="CSS logo">
+                <p class="program-name">CSS</p>
+              </li>
+            </ul>
+          </div>
+          <div class="frame-container">
+            <div class="frame-arrow">
+              <h3>Frameworks</h3>
+              <img src="./images/arrow2.png" alt="Arrow right">
+            </div>
+          </div>
+          <div class="line"></div>
+          <div class="skills-container">
+            <div class="skills-arrow">
+              <h3>Skills</h3>
+              <img src="./images/arrow2.png" alt="Arrow right">
+            </div>
+          </div>
+          <div class="line"></div>
+        </article>
```
- [x] Nice work putting the technology in each category(e.g., "JavasScript", "HTML" and "CSS") in a `ul` tag and each of them a `li`. However, also consider putting the technology categories ("Languages", "Frameworks" and "Skills") in a `ul` tag, each of them a `li`. This is to make your code organized by grouping the items in a list together.

julie-ify reviewed (changes requested):
## Hi @steveWDamesJr, 

Good job so far!

Highlight!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️

Hello ✋, you have done well reaching this point of the milestone, Your project is great and it matches with the design from your chosen template🤝. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

steveWDamesJr committed:
Fix semantics in markup

Change article tags to sections as per reviewer requests
Update style selector to reflect changes

steveWDamesJr committed:
Update markup semantics section tag

rustam-tolipov reviewed (approved):
## Hi @steveWDamesJr 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

![image](https://user-images.githubusercontent.com/69011963/133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7.png)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
